app/app.js

```diff
@@ -8,6 +9,41 @@ import config from "timed/config/environment";
 import "simplebar";
 import "simplebar/dist/simplebar.css";
 
+if (config["@sentry/ember"]) {
+  const sentryConfig = config["@sentry/ember"].sentry;
+  Sentry.init({
+    dsn: sentryConfig.dsn,
+    ignoreErrors: config.SENTRY_IGNORE,
+    tracesSampleRate: 1.0,
```

Comment:
```suggestion
    tracesSampleRate: 0.01,
```
Please reduce this, since we have to pay for transactions and we just need a tiny fraction of them to gather some insights into performance.